aiohttp_sse/__init__.py

```diff
@@ -149,49 +166,86 @@ def last_event_id(self) -> Optional[str]:
 
         return self._req.headers.get(self.DEFAULT_LAST_EVENT_HEADER)
 
+    @property
+    def ping_interval(self) -> Union[int, float]:
+        """Time interval between two ping massages"""
+        return self._ping_interval
+
     @ping_interval.setter
-    def ping_interval(self, value):
+    def ping_interval(self, value: Union[int, float]) -> None:
         """Setter for ping_interval property.
 
         :param int value: interval in sec between two ping values.
         """
 
-        if not isinstance(value, int):
-            raise TypeError("ping interval must be int")
+        if not isinstance(value, (int, float)):
```

Comment:
Wait, are we adding new support for float? If we want that, we should do it in a separate PR.

Reply:
In the end of the usage is asyncio.sleep() arg, that originally typed as **float**, but also supports int 

I can't imagine why we should strict basic input type of asyncio.sleep arg